doit/task.py

```diff
@@ -407,7 +407,7 @@ def check_attr(task, attr, value, valid):
         @param valid (list): of valid types/value accepted
         @raises InvalidTask if invalid input
         """
-        if isinstance(value, valid[0]):
+        if isinstance(value, tuple(valid[0])):
```

Comment:
why need to "convert" to tuple. they are all tuples already, right?

Reply:
oh. i see. problem is in tests itself. not in the code here...

Reply:
You're right! I apologize, I did the change in a rush while at work. I think the tests should be changed to use tuples and revert the convertion to tuple here, don't you?

Do you prefer to keep cumulating commits or should I rebase and `push -f`?